include/render/pixel_format.h

```diff
@@ -0,0 +1,32 @@
+#ifndef RENDER_PIXEL_FORMAT_H
+#define RENDER_PIXEL_FORMAT_H
+
+#include <wayland-server-protocol.h>
+
+struct wlr_pixel_format_info {
+	uint32_t drm_format;
+
+	/* Equivalent of the format if it has an alpha channel, 0 if NA */
+	uint32_t opaque_substitute;
+
+	/* Bytes per pixels */
+	uint32_t bpp;
+
+	/* Size of each channels, 0 if NA */
+	struct {
```

Comment:
I don't know if that's useful, but I thought it would be good to have it. I'm open to remove it if that's really useless.